Engine/src/Engine/ECS/ECS.h

```diff
@@ -0,0 +1,19 @@
+#pragma once
+#include "pch.h"
+#include "ECS_Utility.h"
+#include "SparseArray.h"
```

Comment:
same as before, should remove file if not used

Reply:
This is the main header file used by others to access the ecs functionality

Reply:
ohhh ok then just put a note that we just need to include this file